From 65381221fca3a4bba17bd54054ad542a64ee3899 Mon Sep 17 00:00:00 2001
From: Lee Duncan <lduncan@suse.com>
Date: Wed, 24 Jun 2020 11:07:53 -0700
Subject: [PATCH 1/6] Fix compiler complain about multiple defs of my_stdout.

A simple fix: define it in one place, refer to it
elsewhere.
---
 test/test-defs.h | 2 +-
 test/test.c      | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/test/test-defs.h b/test/test-defs.h
index 28af1a5521c1..3378373db2e2 100644
--- a/test/test-defs.h
+++ b/test/test-defs.h
@@ -42,7 +42,7 @@
 #define inval_path 		"/sys/invalid/path"
 #define FUNC_TABLE_SIZE  	(sizeof(func_table) / sizeof(void *))
 
-FILE *my_stdout;
+extern FILE *my_stdout;
 
 #define dbg_print(format, arg...) fprintf(my_stdout, format, ## arg)
 
diff --git a/test/test.c b/test/test.c
index 74db894ee19c..ab2397d1c847 100644
--- a/test/test.c
+++ b/test/test.c
@@ -27,6 +27,8 @@
 #include "test-defs.h"
 #include <errno.h>
 
+FILE *my_stdout;
+
 /*************************************************/
 char *function_name[] = {
 	"sysfs_get_mnt_path",
-- 
2.26.2

